### DISCUSSION TITLE
Light Weight Floating Tabs / Windows

### DISCUSSION THREAD

**Discussion by sirinath** [zen-browser/desktop#2628] [Ideas]
Can we have the ability to pop out a tab to making it a floating tab.

Also arrange these tabs on the screen. This is different split view. There are multiple browser tab windows which mix with other application windows on the Desktop. They can have frame or not based on settings.

**Comment by freddy1301** [comment]
I also wanted to recommend this. A new Window without its own Menus just for the Tab would be perfect. And then keep the Tab in the Tabbar of the Main Window and gray it out, show an icon or somthing like that
